Ryujinx/Ui/GLRenderer.cs

```diff
@@ -0,0 +1,489 @@
+        {
+            WaitEvent = new ManualResetEvent(false);
+
+            _device = device;
+
+            this.Initialized += GLRenderer_Initialized;
+            this.Destroyed += GLRenderer_Destroyed;
+
+            Initialize();
+
+            _chrono = new System.Diagnostics.Stopwatch();
+
+            _ticksPerFrame = System.Diagnostics.Stopwatch.Frequency / TargetFps;
+
+            _primaryController = new Input.NpadController(ConfigurationState.Instance.Hid.JoystickControls);
+
+            AddEvents((int)(Gdk.EventMask.ButtonPressMask |
+                          Gdk.EventMask.ButtonReleaseMask |
+                          Gdk.EventMask.PointerMotionMask | 
+                          Gdk.EventMask.KeyPressMask      |
+                          Gdk.EventMask.KeyReleaseMask));
+        }
+
+        private void Parent_FocusOutEvent(object o, Gtk.FocusOutEventArgs args)
+        {
+            IsFocused = false;
+        }
+
+        private void Parent_FocusInEvent(object o, Gtk.FocusInEventArgs args)
+        {
+            IsFocused = true;
+        }
+
+        private void GLRenderer_Destroyed(object sender, EventArgs e)
+        {
+            Exit();
+
+            this.Dispose();
+        }
+
+        public void HandleScreenState(KeyboardState keyboard)
+        {
+            bool toggleFullscreen = keyboard.IsKeyDown(OpenTK.Input.Key.F11) ||
+                ((keyboard.IsKeyDown(OpenTK.Input.Key.AltLeft) || keyboard.IsKeyDown(OpenTK.Input.Key.AltRight)) 
+                && keyboard.IsKeyDown(OpenTK.Input.Key.Enter));
+
+            if (toggleFullscreen == _toggleFullscreen)
+            {
+                return;
+            }
+
+            _toggleFullscreen = toggleFullscreen;
+
+            if (IsFocused)
+            {
+                if (this.ParentWindow.State.HasFlag(Gdk.WindowState.Fullscreen) )
+                {
+                    if (keyboard.IsKeyDown(OpenTK.Input.Key.Escape) || _toggleFullscreen)
+                    {
+                        this.ParentWindow.Unfullscreen();
+                        (this.Toplevel as MainWindow)?.ToggleExtraWidgets(true);
+                    }
+                }
+                else
+                {
+                    if (keyboard.IsKeyDown(OpenTK.Input.Key.Escape))
+                    {
+                        Exit();
+                    }
+
+                    if (_toggleFullscreen)
+                    {
+                        this.ParentWindow.Fullscreen();
+                        (this.Toplevel as MainWindow)?.ToggleExtraWidgets(false);
+                    }
+                }
+            }
+        }
+
+        private void GLRenderer_Initialized(object sender, EventArgs e)
+        {
+            // Release the GL exclusivity that OpenTK gave us.
+            GraphicsContext.MakeCurrent(null);
+
+            WaitEvent.Set();
+        }
+
+        protected override bool OnConfigureEvent(EventConfigure evnt)
```

Comment:
```suggestion
        protected override bool OnConfigureEvent(EventConfigure event)
```
And same in others functions

Reply:
event is a reserved word